hello_world.py

```diff
@@ -6,3 +6,11 @@ def hello_world():
 hello_world()
```

Comment:
Reporter: FLAKE8
Rule: 0111
Severity: INFO
File: hello_world.py L1
 

Missing module docstring

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>